src/Marker.js

```diff
@@ -44,7 +45,12 @@ class MapMarker extends Component {
   }
 
   render() {
-    if (!this.props.children) return <DefaultMarker {...this.props} />;
+    let hasOnlyCalloutChildren = true;
+
+    React.Children.forEach(this.props.children, child => {
```

Comment:
I think you should use `reduce` like below.
```
let hasOnlyCalloutChildren = this.props.children.reduce(child =>{}, true)
```

Reply:
On `React.Children` , `.reduce` is not implemented...

https://ja.reactjs.org/docs/react-api.html#reactchildren

Reply:
Use `React.Children.toArray.reduce`

```js
const hasOnlyCalloutChildren = React.Children.toArray(this.props.children).reduce(
  (acc, child) => {},
  true
);
```